diff --git a/node_modules/hpack/lib/dynamic_table.js b/node_modules/hpack/lib/dynamic_table.js
index c1663b2..8c7f7f1 100644
--- a/node_modules/hpack/lib/dynamic_table.js
+++ b/node_modules/hpack/lib/dynamic_table.js
@@ -1,4 +1,4 @@
-var util = require('util');
+var util = require('node:util');
 var DynamicTable = module.exports = function () {
     this.table = [[void(0), void(0), void(0)]];
     this.maxTableSize = 4096;
